tasks/create-advisory/README.md

```diff
@@ -18,6 +18,12 @@ Only all `redhat-pending` or all `redhat-prod` repositories may be specified in
 | synchronously            | Whether the task should wait for InternalRequests to complete                             | Yes      | true                        |
 | pipelineRunUid           | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       |                             |
 
+## Changes in 4.4.0
+- Update task to support new repository change. 
+- The require quay.io repsoitory value will be available in snapshot repository key.
```

Comment:
`The require quay.io`? huh?
`s/repsoitory/respository/`

Reply:
Fixed, my bad.